docs/content/en/docs/configuration/command-line.md

```diff
@@ -152,6 +195,21 @@ Disables in memory pod list and also pod watch for changes. Pod list and watch i
 
 ---
 
+## --election-id
+
+The ID to be used for electing ingress controller leader.  Defaults to `ingress-controller-leader`.
+
+---
+
+## --force-namespace-isolation
+
+Whether to force namespace isolation.  This flag is required to avoid the reference of secrets,
+configmaps or the default backend service located in a different namespace than specified in the
+flag `--watch-namespace` (which defaults to all namespaces, so you will probably want to set that
+flag, too).
+
```

Comment:
Perhaps mention the need to set `--disable-node-list`, too.

Reply:
No need to add it, just deprecated the `--disable-node-list` option.